tslint.json

```diff
@@ -2,9 +2,6 @@
   "extends": ["tslint-config-airbnb", "tslint-config-prettier"],
   "rulesDirectory": ["src/", "bin/"],
   "rules": {
-    "max-line-length": {
-      "options": [100]
```

Comment:
Why did you removed that?

Reply:
Because of my comment which was too long
https://github.com/blablacar/ui-library/pull/477/files#diff-fba6813c303386acd94968eaa935dcf1R42
Though it's not totally removed since prettier won't let you go above 100 characters, but it's handling it better than tslint.